src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
```

Comment:
Consider adding a brief description for each test case as a comment to make the code more readable 📚